app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    # cache stores in format [[data-1, result-1] ... [data-n, result-n]]
+    results = []
+
+    def wrapper(*args, **kwargs) -> int:
+        # check is the input data is present in cache
+        if args in [_[0] for _ in results]:
```

Comment:
_ is bad name for variable

Reply:
But in video lessons speaker named underscores as garbage variables. And he told that it is good way to store temporary data

Reply:
We can use underscore nane for variables if we don't use them in code